client/src/styles.scss

```
@@ -289,10 +289,9 @@ button.mat-mdc-menu-item.selected {
     font-size: 16px;
 }
 
-.queue {
-    .mat-expansion-panel-body {
-        padding: 0 !important;
-    }
+.mat-expansion-panel-body {
+    padding: 0 !important;
+    margin-bottom: 16px;
```

I am not sure if this change was intended. If not please revert.  

```suggestion
.queue {
    .mat-expansion-panel-body {
        padding: 0 !important;
    }
```

---

I also meant the removal of the wrapping `.queue`.